Sources/web3swift/Tokens/ERC1376/Web3+ERC1376.swift

```diff
@@ -92,6 +87,9 @@ public class ERC1376: IERC1376, ERC20BaseProperties {
         self.transaction = transaction
         self.transaction.to = address
         self.abi = abi
+        // Forced because this should fail if contract is wrongly configured
+        let contract = web3.contract(Web3.Utils.erc20ABI, at: address)!
```

Comment:
Use `abi` instead of `Web3.Utils.erc20ABI`.

Reply:
good catch! copy and paste 😅